Loop54.Shared/Model/EntityAttribute.cs

```diff
@@ -11,6 +11,9 @@ namespace Loop54.Model
     /// </summary>
     public class EntityAttribute
     {
+        /// <summary>Attribute names are treated as case-insensitive by the engine, though the case received should be preserved preserved.</summary>
```

Comment:
This comparer is used in several places now, so added a static field for it, same as in the engine

Reply:
What's the reason for this? To make the call sites less verbose? Or to be able to change it everywhere by only changing it here?

Reply:
It's not so likely that we'll change it, but it's more to make it less likely that we'll use the wrong one. So at each call site rather than asking "is it using the correct case of comparer" the reviewer only needs to ask "is it using EntityAttribute.NameComparer for an attribute name". This also allows customers to use it and be consistent with how we compare attribute names.

Reply:
Makes sense